nix/sources.json

```diff
@@ -84,6 +84,18 @@
         "url": "https://github.com/informalsystems/ibc-rs/archive/daad02843a091bbdb3dd608e5f4ce790895c8845.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
+    "relayer": {
+        "branch": "master",
+        "description": "An IBC relayer for ibc-go",
+        "homepage": "https://github.com/mmsqe/relayer",
```

Comment:
replace with the fork after merged: https://github.com/crypto-org-chain/relayer/